app/styles/custom-components/_vlc-login.scss

```diff
@@ -0,0 +1,12 @@
+.vlc-login {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.vlc-login-content {
+  margin: auto auto;
```

Comment:
This can be just `margin: auto`. `auto auto`, or any double-value assignment to these spacing properties allow you to set different spacing for the x and y spacings, but here you make them both `auto`, which equals to `margin: auto`.